pyspecdata/core.py

```diff
@@ -7413,99 +7369,6 @@ def bootstrap(self,points,swap_out = exp(-1.0),seedval = 10347,minbounds = {},ma
                         recordlist[runno][name] = thiscopy.output(name)
         print(r'\end{verbatim}')
         return recordlist # collect into a single recordlist array
-    def guess(self):
-        r'''provide the guess for our parameters; by default, based on pseudoinverse'''
```

Comment:
I would not delete this -- I actually think it could be quite useful for this dictionary of guesses that we are talking about.